lib/internal/dns/utils.js

```diff
@@ -184,6 +186,29 @@ function emitInvalidHostnameWarning(hostname) {
   );
 }
 
+let defaultVerbatim;
+
+function getDefaultVerbatim() {
+  if (defaultVerbatim === undefined) {
+    const option = getOptionValue('--dns-order');
+    switch (option) {
+      case 'verbatim':
+        defaultVerbatim = true;
+        break;
+      case 'ipv4':
+      default:
+        defaultVerbatim = false;
+    }
+  }
+
+  return defaultVerbatim;
+}
+
+function setDefaultVerbatim(value) {
+  validateBoolean(value, 'dns.verbatim');
+  defaultVerbatim = value;
+}
```

Comment:
Should we allow passing `undefined` to use the value defined by the CLI flag?

Reply:
Reasonable. But passing `undefined` is also a bit tricky, e.g.:
```js
require('dns').verbatim = undefined
assert(require('dns').verbatim === undefined) 
// assert will throw as getting verbatim make it changed to true/false again
```
Other similar properties, like: `http.globalAgent.maxSockets`, also don't support to restore to default value. I would like to reconsider this if we have more concrete use case.